docs/getting-started/using-testnet.md

```diff
@@ -0,0 +1,46 @@
+# Using Testnet Guide
+
+last updated for v1.0.0/testnet2
+
+---
+
+## Why Use Testnet?
+
+Testnet2 is a wonderful place where you can experiment with the Decred applications without worrying that a mistake will cost you real money. It is actually recommended that people use the Testnet2 to learn the basics of the Decred software or any new features.
```

Comment:
As you can see I've been commenting as I go along and you do have that blurb right below :)

I still think we should generally call it testnet instead of testnet2.